### ISSUE TITLE
unittest for pickling

### ISSUE THREAD

**Issue by mmaelicke** [mmaelicke/scikit-gstat#47]
@redhog, As the `Variogram` can be pickled with #45, I think there should also be a unittest, so that future changes do not break this great feature anymore. Do you have some code snippets lying around from development that you might share?

**Comment by redhog** [comment]
Untested, but the following should be a pretty good test:

```
from skgstat import Variogram, DirectionalVariogram, OrdinaryKriging
import pandas as pd
import numpy as np
import pickle

data1 = pd.read_csv('data/sample_matern_15.csv')
variogram = DirectionalVariogram(data1[['x', 'y']].values, data1.z.values, normalize=False)
reloaded_variogram = pickle.loads(pickle.dumps(variogram))
```

**Comment by mmaelicke** [comment]
thanks, implemented a simple test with 8d54011dda9ce6ce8980dc1f6a2df3b6ed5b03dd
